### ISSUE TITLE
Dark Shade on Blocks with Shaders

### ISSUE THREAD

**Issue by Syinite** [CreativeMD/LittleTiles#209]
When using shaders, and I hit a block, and chisel it, it becomes darker. This happens with anyblock, and I have tried many shaders. This doesn't happen with C&B, but id prefer to use LittleTiles. Is this a known issue, or is there a workaround i'm missing?

![2019-07-24_02 20 15](https://user-images.githubusercontent.com/50846784/61773675-e2141500-adba-11e9-941a-a3ce93c6f8b4.png)

Here is without

![2019-07-24_02 22 48](https://user-images.githubusercontent.com/50846784/61773698-ef310400-adba-11e9-93e5-74e4269b7d19.png)

**Comment by CreativeMD** [comment]
Actually this is a known issue for a long time and it only occurs with certain shaders. I always thought it was the shader's fault, but it's actually LT doing something wrong. Your little hint that it works with c&b helped me a lot. I contacted the author of the BSL Shader who could tell me where the issue is. I had to do some rework and now it finally works ... this issue has been around for far too long. Thanks for reporting this issue, new version will be out soon!
